source/content/guides/integrated-composer/02-dependencies.md

```diff
@@ -19,16 +19,18 @@ It's important to understand how `require` and `require dev` are used on the Pan
 
 ### Composer `require` Section
 
-You should use the `require` section of your `composer.json` file for dependencies your web project needs, even dependencies only used on non-Live environments. Drupal modules / themes and WordPress plugins / themes should always be in the `require` section, not the `require-dev` section. All dependencies in the `require` section are pushed to Pantheon Dev and Multidev environments, but not to Test and Live environments.
-
-You do not need to pass the `--no-dev` option to prevent dependencies from being pushed to your Test and Live environments. This option is passed automatically for Drupal modules / themes and WordPress plugins / themes in the `require` section.
+You should use the `require` section of your `composer.json` file for dependencies your web project needs, even dependencies only used on non-Live environments. Drupal modules / themes and WordPress plugins / themes should always be in the `require` section, not the `require-dev` section. Dependencies in the `require` section are pushed to all Pantheon environments.
```

Comment:
> even dependencies only used on non-Live environments

is this true?  Shouldn't dependencies for only-non-Live environments be excluded from `require` and only be in `require-dev`?

Reply:
The first sentence is unclear. **Modules, plugins and themes** should always be in the `require` section, not the `require-dev` section, as it can be a mess if a module "disappears" when still enabled in the database. The second sentence clarifies this. Perhaps the first sentence should not use the word "dependencies".

Reply:
Thanks @greg-1-anderson and @namespacebrian I removed the unclear sentence.